ios/ReactNativeUsabilla.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It introduces a new file?

Reply:
Yes, `Xcode 9.3` introduced this new file to store the state of necessary workspace checks.
you can read more in the [Xcode 9.3 release notes](https://developer.apple.com/library/content/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-DontLinkElementID_7)